src/server/api/routers/subscription.ts

```
@@ -4,6 +4,7 @@ import { z } from "zod";
 import { logger } from "~/_utils";
 import { env } from "~/env";
 import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+import { getPriceIdForCheckout, getPricingForRole } from "~/server/_utils/pricing";
```

ESLint

[error] `~/server/_utils/pricing` import should occur before import of `~/server/api/trpc`

(import/order)